src/db/mysql/schema.rs

```diff
@@ -10,11 +12,14 @@ table! {
 
 table! {
     bso (user_id, collection_id, id) {
+        #[sql_name="userid"]
         user_id -> Integer,
+        #[sql_name="collection"]
         collection_id -> Integer,
         id -> Varchar,
         sortindex -> Nullable<Integer>,
         payload -> Mediumtext,
+        #[sql_name="modified"]
```

Comment:
kill?

Reply:
Probably, mostly there because I was debating that we should be consistent about using `last_modified` as the variable name.